account-kit/smart-contracts/src/ma-v2/actions/install-validation/installValidation.ts

```diff
@@ -0,0 +1,160 @@
+import {
+  AccountNotFoundError,
+  IncompatibleClientError,
+  isSmartAccountClient,
+  EntityIdOverrideError,
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'EntityIdOverrideError' is defined but never used.
<!-- __reviewdog__:ChA3YzU5NTQzMTdhNjk3ZGYyEgZyZGpzb24= -->